CONTRIBUTING.md

```diff
@@ -47,7 +47,7 @@ By default, the `simple-phpunit` binary chooses the correct PHPUnit version for
 the PHP version you are running. To run tests against a specific PHPUnit
 version, use the `SYMFONY_PHPUNIT_VERSION` environment variable:
 
-```
+```console
 $ SYMFONY_PHPUNIT_VERSION=7.5 vendor/bin/simple-phpunit
```

Comment:
I realize this is outside the PR scope, but it's a good opportunity to bump this to a later version. Given https://phpunit.de/supported-versions.html, I don't think anything older than 8.5 would be used in practice.

Alternatively, we can remove this example and just link to https://symfony.com/doc/current/components/phpunit_bridge.html#modified-phpunit-script for canonical documentation.